pages/docs/installation/installation-distribution.md

```diff
@@ -14,6 +14,39 @@ publishing here lists of known-to-work versions.
 
 Releases of the preCICE distribution are irregular. The version of each distribution is `yymm.r`, reflecting the year, the month, and the revision (bugfixes) of the distribution. Bindings versions reflect compatibility with a specific preCICE version, while adapters use a completely independent versioning scheme. The tutorials follow a `yyyymm.r` scheme and are targeting the released versions of each component. The VM version is based on the tutorials version, followed by the VM revision. While the distribution uses two year digits for convenience, the tutorials and the VM use four digits to allow version comparisons with previous releases that already used four digits.
 
+## v2404.0
+
+(Publication on DaRUS pending)
+
+This is the first release that includes preCICE v3.x.
+
+It comprises the following components:
+
+- preCICE: [v3.1.1](https://github.com/precice/precice/releases/tag/v3.1.1)
+- Tools:
+  - ASTE: [v3.1.0](https://github.com/precice/aste/releases/tag/v3.1.0)
+  - config-visualizer: [v0.2.0](https://github.com/precice/config-visualizer/releases/tag/v0.2.0)
+  - config-visualizer-gui: commit [ae202e0](https://github.com/precice/config-visualizer-gui/tree/ae202e06ade780e02f31c8a88bb62de2c812ecdb) (new)
```

Comment:
There are newer releases for both.

https://github.com/precice/config-visualizer/tags
https://github.com/precice/config-visualizer-gui/tags

The config-visualizer needs another bugfix

Reply:
```suggestion
  - config-visualizer: [v1.1.2](https://github.com/precice/config-visualizer/releases/tag/v1.1.2)
  - config-visualizer-gui: [v0.1.0](https://github.com/precice/config-visualizer-gui/releases/tag/v0.1.0) (new)
```

Reply:
Why are these tags but not GitHub releases? This is confusing.

Reply:
Done